test/example/stack-utils.cpp

```diff
@@ -221,79 +221,194 @@ void test_signature_composition() {
   }
 }
 
-void test_signature_satisfaction() {
-  std::cout << ";; Test stack signature satisfaction\n";
-  // No unreachable
+void test_signature_subtype() {
+  std::cout << ";; Test stack signature subtyping\n";
+  // Differences in unreachability only
   {
-    StackSignature a{Type::i32, Type::f32, false};
-    Signature b(Type::i32, Type::f32);
-    assert(a.satisfies(b));
+    StackSignature a(Type::none, Type::none, true);
+    StackSignature b(Type::none, Type::none, false);
+    assert(StackSignature::isSubType(a, b));
+    assert(!StackSignature::isSubType(b, a));
   }
+  // Covariance of results
   {
-    StackSignature a{Type::i32, Type::f32, false};
-    Signature b({Type::i64, Type::i32}, {Type::i64, Type::f32});
-    assert(a.satisfies(b));
+    StackSignature a(Type::none, Type::funcref, false);
+    StackSignature b(Type::none, Type::anyref, false);
+    assert(StackSignature::isSubType(a, b));
+    assert(!StackSignature::isSubType(b, a));
   }
+  // Contravariance of params
   {
-    StackSignature a{Type::i32, Type::f32, false};
-    Signature b({Type::i64, Type::i32}, {Type::i64, Type::i64, Type::f32});
-    assert(!a.satisfies(b));
+    StackSignature a(Type::anyref, Type::none, false);
+    StackSignature b(Type::funcref, Type::none, false);
+    // assert(StackSignature::isSubType(a, b));
+    // assert(!StackSignature::isSubType(b, a));
```

Comment:
Why are these disabled? We can't compute GLB now, but we don't need to compute it in `isSubType`.

Reply:
Good catch, thanks!